xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -439,6 +439,19 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
   if (m_streaminfo)
   {
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_AUDIOOUTPUT_SUPPORTSDTSHDCPUDECODING))
```

Comment:
It only makes sense if user decides to use dcadec ... if not the streaminfo is not what he is actually getting as plain dts is decoded and therefore 6 channels have to be displayed ...